layers/+tools/command-log/packages.el

```diff
@@ -14,7 +14,7 @@
       ;; is an open PR to merge the fork into the original repo.
       ;; TODO when the PR is merged upstream, change to use the original package
       ;; from MELPA (IOW remove :location argument)
-      '((command-log-mode :location (recipe :fetcher github
+      '((command-log-mode :location (command-log-mode :fetcher github
```

Comment:
I thought that packages with github fetcher are not marked as outdated every time. Does they?